wiki/en/misc/1-index.html

```diff
@@ -30,26 +30,25 @@ <h2>What is Jamulus?</h2>
 </div>
 <div class="fx-row fx-row-center-xs">
   <div class="fx-col-100-xs fx-col-50-l">
-     <h2>Jamulus worldwide</h2>
-    All over the world Jamulus allows huge choirs to rehearse, loud rock bands to play, and brings folk and classical musicians together. It's being used for remote music lessons,
-    in schools and universities, in private and in public. Even just for the spoken word – all in real time on the Internet, as if you were there in person.
-    <h2>Help needed?</h2>
+     <h2>Worldwide</h2>
+    Huge choirs, rock bands, folk and classical musicians, remote music lessons, in schools and universities,
+    private or public. Even the spoken word — as if you were there in person.
+    <h2>Help?</h2>
     <p>
-      Have a look at the <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">documentation</a>. Especially consider the <a href="wiki/Client-Troubleshooting"
-        target="_blank" rel="noreferrer">troubleshooting section</a>!
+      Check out the <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">documentation</a>. Especially the <a href="wiki/Client-Troubleshooting"
+        target="_blank" rel="noreferrer">troubleshooting</a>!
       You can also ask on the <a href="https://github.com/jamulussoftware/jamulus/discussions" target="_blank" rel="noreferrer">forums</a>.
     </p>
   </div>
   <div class="fx-col-100-xs fx-col-50-l">
-    <h2>Want to get involved?</h2>
+    <h2>Get involved</h2>
     <p>
-    Ideas? Found a bug? Want to contribute some code or help translating into your language? Since Jamulus is <a href="https://www.gnu.org/philosophy/free-sw.en.html" target="_blank" rel="noreferrer" title="What is free software?">free and open source software</a> (FOSS) licensed under the <a href="https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html" title="GNU General Public License, version 2" target="_blank" rel="noreferrer">GPL</a>, you can help us!<br/>
-    Take a look at our <a href="wiki/Contribution">contribution guidelines</a> to find out how. Everybody is welcome!
+    Use, see, change, and share — with all. <a href="https://github.com/jamulussoftware">write code</> or <a href="https://hosted.weblate.org/engage/jamulus/">translate it on Weblate</a>.<br/>
```

Comment:
I think this should stay roughly similar to what it was and not mostly link to Weblate.

Reply:
It halfway links to GitHub, and halfway links to Weblate.
What part that went missing did anything useful?
The link to GPLv3 I understand (and I want to put it next to "copylefted libre" above), the rest I don't.

Reply:
> I think there should be a Weblate link in icon form at the bottom

Probably yes. The Weblate logo should be svg and use the same color as the other ones.
> It halfway links to GitHub, and halfway links to Weblate.

But it doesn't explicitly say "on GitHub"

> The link to GPLv3 I understand 

Exactly. I think it should be linked prominently in the contributing paragraph as mostly developers (or users with coding background should know about it).

> What it is in the capacity of ensuring freedom for use is important (higher up). How one can additionally contribute is a secondary concern, is my thinking.

I think it gets philosophic here. I think contribution and the basic (4) rights in spirit of the Free Software foundation are very closely linked, at least in practice.

Reply:
Yes, but the sequence in which things are presented is important. Just service design.
Nothing good is presented as "no cost" up front.
Similarly, there is more to the use of Jamulus than its continuation or possible improvement.
One has to sell people on using Jamulus after they understand what it is, and can then contribute knowing this.